Orange/widgets/data/owmergedata.py

```diff
@@ -212,10 +213,36 @@ def group_table_indices(table, key_vars, exclude_unknown=False):
     return groups
 
 
+def group_table_indices_value(table, key_var, exclude_unknown=False):
+    groups = defaultdict(list)
+    for i, inst in enumerate(table):
+        key = inst[key_var]
+        if exclude_unknown and math.isnan(key):
+            continue
+        groups[key.value].append(i)
+    return groups
+
+
 def left_join_indices(table1, table2, vars1, vars2):
+    indices = []
+    var1, var2 = vars1[0], vars2[0]
+    if isinstance(var1, TimeVariable) and isinstance(var2, TimeVariable):
```

Comment:
Is this a general solution? Would everyone merging on time variables consider this behavior sensible?